doc/docs/master/getting_started/install-pachctl-completion.md

```diff
@@ -0,0 +1,107 @@
+# Install `pachctl` Autocompletion
+
+Pachyderm autocompletion allows you to automatically finish
+partially typed commands by pressing `TAB`. Autocompletion needs
+to be installed separately when `pachctl` is already
+available on your client machine.
+
+Pachyderm autocompletion is supported for `bash` and `zsh` shells.
+You must have either of them preinstalled
+before installing Pachyderm autocompletion.
+
+!!! tip
+    Type `pachctl completion --help` to display help information about
+    the command.
+
+## Install `pachctl` Autocompletion for `bash`
+
+If you are using bash as your preferred shell, follow the steps in this
+section to install `pachctl` completion for bash.
+
+To install `pachctl` autocompletion for `bash`, perform the following steps:
+
+1. Verify that `bash-completion` is installed on your machine.
+   For example, if you have installed bash completion by using Homebrew,
+   type:
+
+   ```bash
+   brew info bash-completion
+   ```
+
+   This command returns information about the directory in which
+   `bash-completion` and bash completion scripts are installed.
+   For example,  `/usr/local/etc/bash_completion.d/`. Unless it is
+   the default `/etc/bash_completion.d/` location, you need to specify
```

Comment:
`/etc/bash_completion.d/` is not the default for homebrew-installed `bash-completion`. Technically, bash-completion doesn't have a default, although `/etc/bash_completion.d/` is commonly employed. Given that, I'd just strike out this sentence, or reword it.

Reply:
reworded